src/main/java/hlf/java/rest/client/controller/FabricOperationsController.java

```diff
@@ -126,4 +125,21 @@ public ResponseEntity<ClientResponseModel> getDeserializedJson(
       @RequestParam(name = "prettyPrint", required = true) boolean prettyPrint) {
     return serializationUtil.decodeContents(encodedJson, decodeInterior, prettyPrint);
   }
+
+  /**
+   * Add anchor peer(s) of an organization to a channel. Anchor peer addition should be done once
+   * the peer nodes of the organization have joined the channel.
+   *
+   * @param channelName - the name of the channel for which you wish to add the anchor peer nodes
+   *     to.
+   * @param channelUpdateParamsDTO - contains the details for the organization peers you wish to be
+   *     added to the channel as anchor peer.
+   * @return ResponseEntity<ClientResponseModel> - contains the result of the operation.
+   */
+  @PostMapping(value = "/channel/{channelName}/add_anchor_peer")
+  public ResponseEntity<ClientResponseModel> addAnchorPeersToChannel(
+      @PathVariable @Validated String channelName,
+      @RequestBody @Validated ChannelUpdateParamsDTO channelUpdateParamsDTO) {
```

Comment:
[nitpick/question] Will it make it simple to have one channel update API with different query parameter to do different activities? The other inputs to the API remain the same.

Reply:
How I visual this - two operations that requires channel update are performed at different time, like one is done to add new organization to the channel and the other - to add anchor peers to a channel is done once the peer has joined the channel. It is good to keep them as separate since these are entirely different operation and most importantly, impacting the different sections in the config block.
If we see a need to merge them in the future - by introducing more scenarios that require config block update, we could do it.